### ISSUE TITLE
Move multiple vertices at the same time

### ISSUE THREAD

**Issue by 3nids** [wonder-sk/CadNodeTool#5]
I know we skipped this at the beginning.
Would it be a big effort to bring this?
I think it would be very useful with CAD to move a rectangle edge for instance.
Thoughts?

**Comment by wonder-sk** [comment]
We can make use of the selection functionality (currently used just to delete nodes later) - if user click one of the selected nodes, all nodes would start moving. So not really explicit support for moving of segments, but actually more generic functionality...

**Comment by 3nids** [comment]
Yeah sounds like a good compromise. It is not the most intuitive way, but really powerful as you say.

**Comment by thbaumann** [comment]
Would the described moving of selected vertices also work with different features as described here: https://gis.stackexchange.com/questions/217312/drag-multiple-vertices-of-different-line-features-in-qgis ?

**Comment by wonder-sk** [comment]
@rdbath yes.

By the way, "native" moving of segments has been added earlier this week, so I am renaming this ticket to "move multiple vertices" - as that is what still could be added.

**Comment by thbaumann** [comment]
@wonder-sk great. Any idea when this functionality could be part of the plugin? Is there something I could help with to implement this feature?

**Comment by wonder-sk** [comment]
It is not in my plans right now... that said, a pull request implementing this functionality would be welcome if you would like to help.
